### PR TITLE
Updated UI for milestone reports: split report content and recommendations, added tabs

### LINKED ISSUE #2052: Update UI for milestone reports: data + tabs
Current milestone report layout (vertical):
Summary data + graphs
Suggested actions
Student progress/scores

Change layout to:
Summary data + graphs
2 tabs (Suggested actions, Student progress/scores)

### PR DESCRIPTION
**Pull request by breity**
Milestone reports now support two report template sections: report `content` and teacher `recommendations`. The report now shows the `content` in a section called "Class Report" and the `recommendations` in a section called "Recommendations". (See `"milestones"` in the `project.json` for the unit linked below to view the updated template json format.)

Report layout has also been simplified: Class Report is shown as the main section and now there are two tabs below that allow users to toggled between viewing Recommendations and Student Progress. (If no recommendations are specified in the template json, the Recommendations tab doesn't appear.)

To test:
1. Set up a run with with this unit: [61.zip](https://github.com/WISE-Community/WISE/files/3799392/61.zip)
2. As a student/students, complete some work on steps 2.4 and 2.5.
3. View the milestone reports in the Classroom Monitor as a teacher and verify that the new tabbed layout appears.

Closes #2052.

### CODE REVIEW

**Comment by ekhumalo** [comment]
Here's an updated project with forward slashes replaced in the json.

[61.zip](https://github.com/WISE-Community/WISE/files/3799392/61.zip)

File: src/main/webapp/wise5/classroomMonitor/milestones/milestonesController.js [hunk 1]
```diff
@@ -359,13 +359,9 @@ class MilestonesController {
...
         const report = this.generateReport(projectAchievement);
-        if (report != null) {
-          projectAchievement.generatedReport = this.generateReport(projectAchievement);
-          this.setReportAvailable(projectAchievement, true);
-        } else {
-          delete projectAchievement.generatedReport;
-          this.setReportAvailable(projectAchievement, false);
-        }
+        this.setReportAvailable(projectAchievement, true);
+        projectAchievement.generatedReport = report.content ? report.content: null;
```
Comment (hirokiterashima) [inline src/main/webapp/wise5/classroomMonitor/milestones/milestonesController.js]:
```suggestion
        projectAchievement.generatedReport = report.content ? report.content : null;
```

File: src/main/webapp/wise5/classroomMonitor/milestones/milestonesController.js [hunk 2]
```diff
@@ -390,8 +386,8 @@ class MilestonesController {
       aggregateAutoScores[componentId] = this.calculateAggregateAutoScores(nodeId, componentId, this.periodId);
     }
     const template = this.chooseTemplate(projectAchievement.report.templates, aggregateAutoScores);
-    let templateContent = template.content;
-    if (templateContent != null) {
+    let content = template.content ? template.content : "";
```
Comment (hirokiterashima) [inline src/main/webapp/wise5/classroomMonitor/milestones/milestonesController.js]:
```suggestion
    let content = template.content ? template.content : '';
```

File: src/main/webapp/wise5/classroomMonitor/milestones/milestonesController.js [hunk 3]
```diff
@@ -406,12 +402,16 @@ class MilestonesController {
...
           const categories = JSON.stringify(milestoneCategories).replace(/\"/g, '\'');
           const data = JSON.stringify(milestoneData).replace(/\"/g, '\'');
-          templateContent = templateContent.replace(regex,
+          content = content.replace(regex,
             `$& categories=\"${categories}\" data=\"${data}\"`);
         }
       }
     }
-    return templateContent;
+    const recommendations = template.recommendations ? template.recommendations : "";
```
Comment (hirokiterashima) [inline src/main/webapp/wise5/classroomMonitor/milestones/milestonesController.js]:
```suggestion
    const recommendations = template.recommendations ? template.recommendations : '';
```

**Pull request merged**